.github/workflows/package_all.yaml

```diff
@@ -8,6 +8,10 @@ on:
       memgraph_version:
         description: "Memgraph version to upload as. If empty upload is skipped. Format: 'X.Y.Z'"
         required: false
+      build_type:
```

Comment:
Perhaps change this input field to be a choice from predefined build types so we eliminate errors due to misspelling, the code would go like this
build_type:
  type: choice
  description: "..."
  default: 'RelWithDebInfo'
  options:
  - Release
  - RelWithDebInfo